src/modules/authentication/components/ChangePassword/CSS/ChangePassword.module.css

```diff
@@ -0,0 +1,43 @@
+
+.form {
+    margin-top: 10px;
+    border-radius: 16px;
+    background-color: #315951E5;
```

Comment:
We need to use css variables for colors in our application to make it easier to implement the light/dark mode feature.